### ISSUE TITLE
size returns wrong result for AFArray 

### ISSUE THREAD

**Issue by dpsanders** [JuliaComputing/ArrayFire.jl#75]
``` julia
v = [2 1]'
vv = AFArray(v)
```

ArrayFire reports that 

``` julia
typeof(vv)
ArrayFire.AFArray{Int64,2}
```

but

``` julia
size(vv)
(2,)
```

**Comment by ranjanan** [comment]
The reason this happens is that when ArrayFire gets `[1 2]'`, it gets the raw pointer, and understands it as a single dimensional array. If you write `ndims(vv)`, you will get `1`.  You're right, it would be nice if this were consistent with Base Julia.

**Comment by gaika** [comment]
fixed in #121

**Comment by gaika** [comment]
merged with [v0.1.0](https://github.com/JuliaComputing/ArrayFire.jl/releases/tag/v0.1.0)
